src/item_factory.cpp

```diff
@@ -1200,26 +1202,25 @@ void Item_factory::load_book( JsonObject &jo, const std::string &src )
     }
 }
 
-void Item_factory::load( islot_comestible &slot, JsonObject &jo, const std::string & )
+void Item_factory::load( islot_comestible &slot, JsonObject &jo, const std::string &src )
 {
-    assign( jo, "comestible_type", slot.comesttype );
-    assign( jo, "tool", slot.tool );
-    assign( jo, "charges", slot.def_charges );
-    assign( jo, "quench", slot.quench );
-    assign( jo, "fun", slot.fun );
-    assign( jo, "stim", slot.stim );
-    assign( jo, "healthy", slot.healthy );
-    assign( jo, "parasites", slot.parasites );
+    bool strict = src == "core";
 
-    if( jo.read( "spoils_in", slot.spoils ) ) {
-        slot.spoils *= 600; // JSON specifies hours so convert to turns
-    }
+    assign( jo, "comestible_type", slot.comesttype, strict );
+    assign( jo, "tool", slot.tool, strict );
+    assign( jo, "charges", slot.def_charges, strict, 1L );
```

Comment:
`1L` is the low limit for charges? How so? `islot_comestible::def_charges` is `long`, and `charges` have nothing to do with `volume`.